pyproject.toml

```diff
@@ -1,32 +1,274 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools>=42"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "mage-ai"
-version = "0.8.75-dev"
-description = "DevEx poetry configuration"
-authors = ["Mage"]
+version = "0.9.21"
+description = "Mage is a tool for building and deploying data pipelines."
 readme = "README.md"
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+authors = [
+  {email = "eng@mage.ai"},
+  {name = "Mage"}
+]
+maintainers = [
+  {name = "Mage", email = "eng@mage.ai"}
+]
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: OS Independent"
+]
 
-[tool.poetry.dependencies]
-python = ">=3.8,<4.0"
-
-[tool.poetry.group.dev.dependencies]
-mypy = "*"
-black = "*"
-flake8 = "*"
-pre-commit = "*"
-pytest = "*"
-pytest-cov = "*"
-interrogate = "*"
-isort = "*"
-commitizen = "*"
-ipykernel = "^6.17.1"
-pandas-stubs = "*"
-flake8-bugbear = "*"
-sourcery = "*"
+dependencies = [
+  'aiofiles==22.1.0',
+  'aiohttp',
```

Comment:
how did you come up with this list of dependnencies? `aiohttp` isn't in the original `requirements.txt` or `setup.py`

Reply:
I went through all imports in code. E.g. its used here: https://github.com/mage-ai/mage-ai/blob/f06f6117b5d2d7f676899769de0a017ed216912e/mage_ai/server/api/projects.py#L1